src/style.css

```diff
@@ -1 +1,83 @@
-/* Styles go here */
+:root {
+  --primary-color: #000;
+  --secondary-color: #00acdc;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  color: var(--primary-color);
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
+}
```

Comment:
Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser to read your web document. Use * only for `box-sizing: border-box;`